website/docs/reference/android/portals-plugin.md

```diff
@@ -18,23 +42,154 @@ The [PortalsPlugin](./portals-plugin) class is a Capacitor Plugin that allows fo
 <TabItem value="kt">
 
 ```kotlin
-PortalManager.newPortal("my_portal")
-    .setPlugins(Arrays.asList(PortalsPlugin::class.java))
-    .create()
-``` 
+// String message
+PortalsPlugin.publish("topic", "message content")
+
+// JSON message
+val items = JSONArray()
+items.put("cheese")
+items.put("bacon")
+items.put("eggs")
+
+PortalsPlugin.publish("cart", items)
+```
+
+</TabItem>
+<TabItem value="java">
+
+```java
+// String message
+PortalsPlugin.publish("cart", items);
+
+// JSON message
+JSONArray items = new JSONArray();
+items.put("cheese");
+items.put("bacon");
+items.put("eggs");
+
+PortalsPlugin.publish("cart", items);
+```
+
+</TabItem>
+</Tabs>
+
+#### Parameters
+
+Name | Type | Description
+:------ | :------ | :------
+`topic` | `String` | The topic associated with the message. [Subscribers](./portals-plugin#subscribe) of this topic will receive the message
+`data` | `Any` | A message to send. **Note**: this is added into a JSONObject to send through the Capacitor Bridge and should be a compatible type: JSONObject, JSONArray, String, Boolean, Integer, Long, Double, or null
```

Comment:
@elylucas I know the answer for Android - what i put here. But I think iOS has a similar constraint on what data can be added when publishing a message but I wasn't sure what to put for it in that doc. I think it might be simple types and array/map. Do you know?

Reply:
I thought this had to be a map or an array, and not a primitive value since it gets converted to json. We should double check that